src/react/account/UpdateAccountPolicy.tsx

```diff
@@ -0,0 +1,166 @@
+import FormContainer, * as F from '../ui-elements/FormLayout';
+import React from 'react';
+import { SecondaryText } from '@scality/core-ui';
+import { Box, Button } from '@scality/core-ui/dist/next';
+import { useForm } from 'react-hook-form';
+import { useMutation, useQuery } from 'react-query';
+import CopyButton from '../ui-elements/CopyButton';
+import { useHistory, useParams } from 'react-router';
+import { useIAMClient } from '../IAMProvider';
+import { queryClient } from '../App';
+import { getListPoliciesQuery, getPolicyQuery } from '../queries';
+import { notFalsyTypeGuard } from '../../types/typeGuards';
+import { Clipboard } from '../ui-elements/Clipboard';
+
+type FormValues = {
+  policyName: string;
+  policyDocument: string;
+};
+
+type Props = {
+  accountName: string;
+  policyName: string;
+  policyDocument: string;
+}
+
+const UpdateAccountPolicy = () => {
+  const IAMClient = useIAMClient();
+  const history = useHistory();
+  const { policyName, arn, path, DefaultVersionId, accountName  } = useParams<{ accountName: string, policyName: string, arn: string, path: string, DefaultVersionId: string }>();
+
+  const defaultValues = {
+    policyName: '',
+    policyDocument: '',
+  };
+  const {
+    register,
+    reset,
+    handleSubmit,
+    watch,
+    formState: { isDirty, errors,  },
+  } = useForm<FormValues>({
+    defaultValues
+  });
+
+  const watchAllFields = watch();
+  const policyArn = arn+'/'+path+'/'+policyName;
+  const { data: policyResult, status: policyStatus } = useQuery(getPolicyQuery(policyArn, DefaultVersionId, IAMClient));
+  const updatePolicyMutation = useMutation(
+    ({ policyDocument } : Props) => {
+      return notFalsyTypeGuard(IAMClient).CreatePolicyVersion(
+        policyArn,
+        policyDocument
+      );
+    },
+    {
+      onSuccess: () =>
+        queryClient.invalidateQueries(
+          getListPoliciesQuery(accountName, notFalsyTypeGuard(IAMClient)).queryKey,
+        ),
```

Comment:
after successfully updating the policy I think we should go back to the policy list.